spec/index.html

```diff
@@ -722,19 +723,25 @@ <h3>Properties of simple entailment and satisfiability</h3>
     
     <p>We define the <dfn>set of propositions</dfn> in an interpretation as follows:</p>
 
-	<p class="fact"> The set of propositions in an interpretation I is IPR(I) = {&nbsp;IT(x, y, z)&#65372;x is in IR, y is in IP, z is in IR&nbsp;}; we observe that a proposition is in the extension of <code>rdfs:Proposition</code>. </p>
+	<p class="fact"> The set of propositions in an interpretation I is IPR(I) = {&nbsp;IT(x, y, z)&#65372;x is in IR, y is in IP, z is in IR&nbsp;}. </p>
```

Comment:
It seems there should be some `code` or other formatting applied to some of these words or symbols... but I'm not sure which, so I'm not adding such. I am adding some `&nbsp;`.
```suggestion
	<p class="fact"> The set of propositions in an interpretation I is
	IPR(I)&nbsp;= {&nbsp;IT(x, y, z)&nbsp;&#65372;&nbsp;x&nbsp;is in IR,
	y&nbsp;is in IP, z&nbsp;is in IR&nbsp;}.</p>
```